### ISSUE TITLE
paper-checkbox checked attribute accessibility issue

### ISSUE THREAD

**Issue by uxdesignz** [PolymerElements/paper-checkbox#213]
Expected Results: 
Once the checkbox is checked, the HTML checked attribute is inserted and which should not be. Do not insert the checked attribute as its use is not valid. Use only aria-checked to communicate the state of the checkbox.

Actual Result
Form Labels: The checkbox ARIA form controls have no label defined and an inappropriate attribute. Screen readers may not be able to correctly associate the label with the form field.

**Comment by uxdesignz** [comment]
Any update on this?
